packages/app/src/cli/utilities/app/http-reverse-proxy.ts

```diff
@@ -37,12 +40,17 @@ export async function getProxyingWebServer(
 function getProxyServerWebsocketUpgradeListener(
   rules: {[key: string]: string},
   proxy: Server,
+  stdout?: Writable,
 ): (req: http.IncomingMessage, socket: import('stream').Duplex, head: Buffer) => void {
   return function (req, socket, head) {
     const target = match(rules, req, true)
     if (target) {
       return proxy.ws(req, socket, head, {target}, (err) => {
-        outputWarn(`Error forwarding websocket request: ${err}`)
+        useConcurrentOutputContext({outputPrefix: 'proxy', stripAnsi: false}, () => {
+          const error = err instanceof AggregateError && err.errors.length > 0 ? err.errors[err.errors.length - 1] : err
+          outputWarn(`Error forwarding websocket request: ${error.message}`, stdout)
```

Comment:
What if we concat all the error messages instead of taking the last one?

```suggestion
          const message = err instanceof AggregateError && err.errors.length > 0 ? err.errors.map(e => e.message).join('; ') : err.message
          outputWarn(`Error forwarding websocket request: ${message}`, stdout)
```

Reply:
I initially did something similar, but in this case the errors are basically duplicates:

![Captura de pantalla 2025-07-03 a las 16.18.53.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/nNsubxpSjsPXUKXMmDt6/9da4db54-f0e9-48c4-a65c-377ead2475b8.png)

So I thought is better to just show one, what do you think?